composer.json

```diff
@@ -5,7 +5,7 @@
   "keywords": ["idp", "id broker", "api", "php client"],
   "license": "MIT",
   "require": {
-    "php": ">=7.0",
+    "php": ">=7.4",
```

Comment:
Wish I was more current on my PHP understanding, but are you aware of anything in this library that specifically requires version 7.4?

Reply:
me too, but I think the syntax for typed parameters does